package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```

Comment:
is this file actually required?

Reply:
no, probably caused by accidentally running npm in parent directory